samila/genimage.py

```diff
@@ -1,13 +1,12 @@
 # -*- coding: utf-8 -*-
 """Samila generative image."""
-import random
```

Comment:
This deletion will lead to a major bug!

Example:
```pycon
>>> from samila import *
>>> g = GenerativeImage()
>>> g.generate()
>>> g.data1
[]
```

Because `eval` function needs `random` module in the evaluation phase.

**Notice:** We should cover something like this in our tests (by controlling the output list length)

Reply:
Yes you're right I'll rollback this commit.
At this moment I don't have any tests we can open a related issue.